src/main/java/Parser.java

```diff
@@ -0,0 +1,145 @@
+import java.time.LocalDate;
+
+/**
+ * Object to parse a line of input given by user
+ */
+public class Parser {
+
+    /**
+     * Returns a Command based on user input
+     * @param input user input
+     * @return command based on user input
+     * @throws DukeException exception that occurs when parser does not
+     * recognise user input
```

Comment:
Perhaps you could add a full stop after each sentence in JavaDoc comments? I think this issue was highlighted in the guidelines for Java coding standard. I observed a similar issue in other JavaDoc comments as well.